aws/resource_aws_pinpoint_app_test.go

```diff
@@ -43,23 +90,18 @@ func TestAccAWSPinpointApp_basic(t *testing.T) {
 }
 
 func TestAccAWSPinpointApp_CampaignHookLambda(t *testing.T) {
-	oldDefaultRegion := os.Getenv("AWS_DEFAULT_REGION")
```

Comment:
Pinpoint is [now available](https://aws.amazon.com/about-aws/whats-new/2019/01/amazon-pinpoint-is-now-available-in-three-additional-regions/) in regions other than `us-east-1`.